homeassistant/components/sonos/media_player.py

```diff
@@ -330,10 +331,10 @@ def _is_radio_uri(uri):
 
 
 class SonosDevice(MediaPlayerDevice):
```

Comment:
Should this be refactored `SonosEntity`?

Reply:
Yeah, that makes sense. I was a bit torn because the parent class is a `Device` so I left it alone.

Thanks for looking so closely, I adjusted it now :)